src/containers/importExamples/import_example_simulated_recording.py

```diff
@@ -0,0 +1,27 @@
+import spikeextractors as se
+import numpy as np
+import labbox_ephys as le
+import kachery_p2p as kp
+
+# adjust these values
```

Comment:
This is part of the template, right? An instruction to the end user to customize the python code that's being displayed on the screen? (Not an instruction to us to come up with a different example?)

Reply:
correct